src/raylib.h

```diff
@@ -928,6 +928,8 @@ RLAPI void SetWindowSize(int width, int height);                  // Set window
 RLAPI void *GetWindowHandle(void);                                // Get native window handle
 RLAPI int GetScreenWidth(void);                                   // Get current screen width
 RLAPI int GetScreenHeight(void);                                  // Get current screen height
+RLAPI int GetRenderWidth(void);                                   // Get current render width
+RLAPI int GetRenderHeight(void);                                  // Get current render height
```

Comment:
I forgot to add these on my last PR.

Reply:
Yeah, I was aware, still not sure if expose those functions, I think some users could be confused...